src/components/Footer/index.module.scss

```diff
@@ -0,0 +1,90 @@
+.footer {
+  &__wrapper {
+    max-width: 1140px;
+    width: 100%;
+    margin: 0 auto;
+    padding: 93px 0 105px;
+
+    @media (max-width: 1200px) {
+      padding: 93px 10px 105px;
+    }
+  }
+
+  &__logo {
+    height: 30px;
```

Comment:
do you really need height here? Could this height be adjustable according to content?

Reply:
I don't need it, so I deleted it